Libraries/LibJS/AST.h

```diff
@@ -783,15 +783,23 @@ class TemplateLiteral final : public Expression {
     {
     }
 
+    TemplateLiteral(NonnullRefPtrVector<Expression> expressions, NonnullRefPtrVector<Expression> raw_strings)
+        : m_expressions(expressions)
+        , m_raw_strings(raw_strings)
```

Comment:
Let's move instead of copy the vectors:
```suggestion
        : m_expressions(move(expressions))
        , m_raw_strings(move(raw_strings))
```